stock_move_backdating/models/stock_picking.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    @api.multi
+    def do_transfer(self):
+        result = super(StockPicking, self).do_transfer()
+        for picking in self:
+            # set date_done as the youngest date among the moves
+            dates = picking.mapped('move_lines.date')

```

Comment:
@tafaRU Are move_lines shouldn't be filtered with
```python
picking.move_lines.filtered(lambda m: m.state == 'done').mapped('date')
```
?
You can have cancelled moves in picking

Reply:
This module is intended to be used on the `pickings` in state `available`, when the Operations tab is shown, so I don't understand the reason to filter only the moves done.

Reply:
Let me try to explain what I mean with youngest date with the following example:
date_1 = `2018-06-12` date_2 = `2018-06-13` --> max(date_1, date_2) = date_2 which is the youngest date, isn'it?

Reply:
That's it. Understand